sdk/sdk-view/src/test/java/mil/emp3/api/KMLSTest.java

```diff
@@ -30,23 +27,20 @@
 import mil.emp3.api.enums.KMLSEventEnum;
 import mil.emp3.api.enums.KMLSStatusEnum;
 import mil.emp3.api.events.KMLSEvent;
+import mil.emp3.api.exceptions.EMP_Exception;
 import mil.emp3.api.interfaces.IKMLS;
-import mil.emp3.api.interfaces.IMap;
 import mil.emp3.api.interfaces.IMapService;
 import mil.emp3.api.listeners.IKMLSEventListener;
-import mil.emp3.api.utils.FileUtility;
-import mil.emp3.core.services.kml.KMLSProvider;
-import mil.emp3.core.services.kml.KMLSRequest;
 
-import static org.mockito.Matchers.any;
-import static org.powermock.api.mockito.PowerMockito.when;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
 @RunWith(RobolectricTestRunner.class)
 @PrepareForTest({Color.class, URLUtil.class})
 public class KMLSTest extends TestBaseSingleMap {
 
-    private static String TAG = KMLSTest.class.getSimpleName();
+    private final static String TAG = KMLSTest.class.getSimpleName();
 
-    private static boolean checkStatus = false; // This is turned off when submitting code as there is an issue on Travis.
+    private final static boolean checkStatus = false; // This is turned off when submitting code as there is an issue on Travis.
```

Comment:
Do we have an issue open for this? Sounds like a problem.

Reply:
Not sure, this has been around for a long time before I started working on it.